tests/unit/components/BadTextInput.spec.js

```diff
@@ -0,0 +1,52 @@
+import { shallowMount, createLocalVue } from "@vue/test-utils";
+
+import BadTextInput from "@/components/BadTextInput.vue";
+
+import Vuex from "vuex";
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+
+describe("Component BadTextInput.vue", () => {
+  let underTest;
+  let onChange;
+
+  beforeEach(() => {
+    onChange = jest.fn();
+    underTest = shallowMount(BadTextInput, {
+      propsData: {
+        change: onChange,
```

Comment:
Is this unused? I don't see `change` declared in the component?

Reply:
Looks like this is a dead variable

Reply:
rrrremoved